### PR TITLE
Bug 1783421, updated cluster limit language

### PR DESCRIPTION
**Pull request by ahardin-rh**
This addresses https://bugzilla.redhat.com/show_bug.cgi?id=1783421 for 3.11 docs

### CODE REVIEW

File: admin_guide/manage_nodes.adoc [hunk 1]
```diff
@@ -630,8 +630,8 @@ resolution configuration.
 Default: 90%
 <4> The percent of disk usage before which image garbage collection is never run.
 Lowest disk usage to garbage collect to. Default: 80%
-<5> The Queries per Second (QPS) to use while talking with the Kubernetes API server. 
-<6> The burst to use while talking with the Kubernetes API server. 
+<5> The Queries per Second (QPS) to use while talking with the Kubernetes API server.
```
Comment (lamek) [inline admin_guide/manage_nodes.adoc]: Super minor, but should "Second" be capitalized here?

File: release_notes/ocp_3_11_release_notes.adoc [hunk 2]
```diff
@@ -415,12 +415,12 @@ spec:
 [[ocp-311-scale]]
 === Scale
 
-[[ocp-311-scale-cluster-limits]]
-==== Cluster Limits
+[[ocp-311-scale-cluster-maximums]]
+==== Cluster Maximums
```
Comment (lamek) [inline release_notes/ocp_3_11_release_notes.adoc]: In your other PR "Cluster Maximums" is not capitalized in section headings. Calling this out so you can standardize one way or the other. I'd go no caps.

Comment (lamek) [reply release_notes/ocp_3_11_release_notes.adoc]: There are a handful of headings where I see this below as well.

**Review by lamek — COMMENTED** [review]
Two minor nits, otherwise LGTM.

File: release_notes/ocp_3_11_release_notes.adoc (hunk 2 above)
Comment (ahardin-rh) [reply release_notes/ocp_3_11_release_notes.adoc]: @lamek Thank you! We began applying sentence case headings in 4.x docs. This is for 3.11, so you'll see older standards applied.

File: _topic_map.yml [hunk 3]
```diff
@@ -737,8 +737,8 @@ Topics:
   File: scaling_cluster_metrics
 - Name: Scaling Cluster Monitoring
   File: scaling_cluster_monitoring
-- Name: Cluster Limits
-  File: cluster_limits
+- Name: Cluster Maximums
```
Comment (tkatarki) [inline _topic_map.yml]: Tested Maximums per Cluster sounds better

**Comment by ahardin-rh** [comment]
Thank you!

**Pull request merged**